### PR TITLE
Add Sentry notifications

### PR DESCRIPTION
**Pull request by joaopapereira**
Added Post URL
/hubot/sentry-notify/:room

to receive sentry notification and to publish them in the channel :room

Added basic test Unit test of the function and a E2E test as well

### CODE REVIEW

**Comment by tansaku** [comment]
@joaopapereira thanks for this - and sorry not to have reviewed it yet - is this ready to go from your POV?

**Comment by joaopapereira** [comment]
@tansaku Yes I believe it is

File: scripts/sentry-notifications.coffee [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        rollbar.reportMessageWithPayloadData payload['error'],
+          origin: 'send_slack_message'
+          level: 'error'
+          custom:
+            error: error
+            response: response
+            body: body
+
+  robot.router.post "/hubot/sentry-notify/:room", (req, res) ->
+    # {
```
Comment (tansaku) [inline scripts/sentry-notifications.coffee]: do we need to keep this commented code?

**Comment by tansaku** [comment]
apologies @joaopapereira could you re-open this against staging?

I need up update the contributing doc

**Comment by tansaku** [comment]
green locally for me

File: scripts/sentry-notifications.coffee (hunk 1 above)
Comment (joaopapereira) [reply scripts/sentry-notifications.coffee]: This is not commented code, but the post message that we receive from sentry

**Comment by joaopapereira** [comment]
Just changed the base to staging

File: scripts/sentry-notifications.coffee (hunk 1 above)
Comment (joaopapereira) [reply scripts/sentry-notifications.coffee]: If you want I can add a comment saying what this is or can just remove it all

Comment (tansaku) [reply scripts/sentry-notifications.coffee]: ah, you're saying it's an example of what we receive?  ah good to have then - yeah, add a note to that effect i guess

File: scripts/sentry-notifications.coffee [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# Description:
+#   Posts Hangouts Notifications from AgileVentures to Slack and Gitter.
```
Comment (tansaku) [inline scripts/sentry-notifications.coffee]: should this change?

File: scripts/sentry-notifications.coffee [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+#   "requestify": "*"
+#
+# Configuration:
+#
+# Commands:
+#   post hangout title, link and type to /hubot/hangouts-notify
+#   post hangout title, video_lin and type to /hubot/hangouts-video-notify
+#
+# Author:
+#   sampritipanda
```
Comment (tansaku) [inline scripts/sentry-notifications.coffee]: should this change?

**Comment by tansaku** [comment]
thanks @joaopapereira 

@mtc2013 any thoughts on this one?

File: spec/sentryIntegrationSpec.coffee [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+      username: 'Sentry Bot'
+      icon_url: '',
+      parse: 'full',
+      token: process.env.SLACK_API_TOKEN)
+    .reply(200, {
+      ok: false,
+      error: 'not_authed'
+    })
+
+makeRequest = (body, done) ->
```
Comment (mtc2013) [inline spec/sentryIntegrationSpec.coffee:25]: @tansaku @joaopapereira it appears we don't need this function in this particular file. i confirmed by commenting it out and the specs still pass.

**Review by mtc2013 — CHANGES_REQUESTED** [review]
Please confirm if this works for you.

File: spec/sentryIntegrationSpec.coffee (hunk 4 above)
Comment (mtc2013) [reply spec/sentryIntegrationSpec.coffee:25]: @tansaku @joaopapereira  i'd advise deleting this makeRequest function from this spec.

**Comment by mtc2013** [comment]
Closing this.  Added a new pr integrating the suggested changes here https://github.com/AgileVentures/agile-bot/pull/40

**Pull request closed without merging**